projects/tuist/fixtures/ios_app_with_extensions/Project.swift

```diff
@@ -106,5 +107,13 @@ let project = Project(
             infoPlist: .default,
             sources: "StaticFramework/Sources/**"
         ),
+        Target(
+            name: "AppIntentExtension",
+            platform: .iOS,
+            product: .extensionKitExtension,
+            bundleId: "io.tuist.App.AppIntentExtension",
+            infoPlist: .default,
```

Comment:
App intents extension need this extra key 

```suggestion
            infoPlist: .extendingDefault(with: [
                "EXAppExtensionAttributes": [
                    "EXExtensionPointIdentifier": "com.apple.appintents-extension"
                ]
            ]),
```

I spotted this when manually creating a new app intents extension in Xcode

![app-intents-plist](https://user-images.githubusercontent.com/11914919/214517184-5698002c-ccea-4df9-acb9-6f3a46992b8e.png)